renovate.json

```diff
@@ -1,37 +1,37 @@
 {
-  "extends": ["config:base", ":disableDependencyDashboard", ":preserveSemverRanges"],
-  "assigneesFromCodeOwners": true,
-  "timezone": "Europe/London",
-  "schedule": ["after 9am every weekday", "before 5pm every weekday"],
-  "enabledManagers": ["npm", "nvm", "dockerfile", "docker-compose", "helmv3", "helm-values", "circleci"],
```

Comment:
We now inherit this from https://github.com/ministryofjustice/hmpps-renovate-config/blob/4acc8951af908562c507a76b68ead835193c6a31/node.json#L4